### DISCUSSION TITLE
Upscaling of old games textures

### DISCUSSION THREAD

**Discussion by WapMap** [upscayl/upscayl#691] [Ideas]
Real-ESRGAN is not bad at its job, but I think it would be nice if be added models to work with textures from old games, such as Amerzone: The Explorer's Legacy and Riven. For example this model: https://openmodeldb.info/models/4x-FSDedither . In most cases, Real-ESRGAN just drains everything into a mess.

**Comment by aaronliu0130** [comment]
You can try [converting it](https://github.com/upscayl/upscayl/wiki/Model-Conversion-Guide) and sending https://github.com/upscayl/custom-models a pull request.

**Reply by WapMap** [reply]
There were no errors after converting according to the instructions, but there is no output file after processing by the model. It seems to me that the conversion instruction is outdated.

**Reply by aaronliu0130** [reply]
@NayamAmarshe This one is an x4 ESRGAN model.

**Reply by NayamAmarshe** [reply]
Works fine here. I just converted it to NCNN with ChainNNer and imported to Upscayl directly, without changing the param file.
<img width="1285" alt="image" src="https://github.com/upscayl/upscayl/assets/25067102/260f90c2-792d-4597-83ec-3d5ca06b90eb">

**Reply by aaronliu0130** [reply]
So we shouldn't change the param file?

**Reply by NayamAmarshe** [reply]
Here's the param.
[fsdeth.param.zip](https://github.com/upscayl/upscayl/files/14160013/fsdeth.param.zip)

Sometimes ESRGAN models work without any changes.

**Reply by WapMap** [reply]
I see, thank you for help.
